Monika-After-Story/MonikaModDev
# Windowreacts but they're implemented how they should've been from the start

Windowreacts now use regexp's to check the active window handle content

## Changes:
- `mas_getActiveWindow` has been renamed to `mas_getActiveWindowHandle` to be more specific to what it does
- `_getActiveWindow_Linux` renamed -> `_mas_getActiveWindowHandle_Linux`
- `_getActiveWindow_Windows` renamed -> `_mas_getActiveWindowHandle_Window`
- The `inclusive` rule for checking active window keys has been removed as this can be entirely handled by regex much better
- Windowreacts no longer uses `.lower()`, (#6802). Instead, this must be manually handled using the `(?i)` flag
- `mas_wrs_r34m` actually checks affection for the exp response. It also makes use of the new idle disp. Additionally, it works for more than just one site now.
- WRS Dev topics have been updated accordingly.

## Testing:
- Verify all windowreacts still work
- Verify r34 reaction has appropriate reactions for aff levels (just two levels, below enam, and enam+)
- Verify no crashes
- Verify other language chars work properly now (again ref to #6802)
